jcl/src/java.base/share/classes/module-info.java.extra

```diff
@@ -63,10 +63,12 @@ exports com.ibm.gpu.spi to
 /*[IF CRIU_SUPPORT]*/
 exports openj9.internal.criu to
     openj9.criu;
+exports openj9.internal.security to
```

Comment:
This export doesn't match any package; I think you meant to export `openj9.internal.criu.security`.